.github/workflows/nightly-centos-sgx1.yml

```diff
@@ -97,6 +97,7 @@ jobs:
         sed -i 's/shelter//g' Makefile
         sed -i 's/inclavared//g' Makefile
         sed -i 's/enclave-tls//g' Makefile
+        sed -i 's/rats-tls//g' Makefile
```

Comment:
There is no need to `sed -i 's/rats-tls//g' Makefile` since there is no `rats-tls` in Makefile